ToDo/ToDo/Services/ToDo/IToDoService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using ToDo.DTO;
+using ToDo.DTOs;
+using ToDo.Model;
+
+namespace ToDo.Services.ToDo
+{
+    public interface IToDoService
+    {
+        Task<List<Model.ToDo>> GetTask(Guid userId, FilterRequest filterRequest);
```

Comment:
```suggestion
        Task<List<Model.ToDo>> GetTasks(Guid userId, FilterRequest filterRequest);
```
lấy list thì lúc nào cx số nhiều nhé